### PR TITLE
Add release notes for Edge Bridge iOS and Android

### PR DESCRIPTION
**Pull request by timkimadobe**
## Description

This PR updates the release notes to include the Edge Bridge 1.0.0 release for Android and iOS

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/pages/documentation/release-notes/index.md [hunk 1]
```diff
@@ -4,6 +4,34 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
...
 
+## October 25, 2022
+
+### iOS AEPEdgeBridge 1.0.0
+
+The `Adobe Experience Platform Mobile SDK - Edge Bridge` extension is now available for iOS!
+
+This extension enables forwarding of Analytics track events to Adobe Edge Network when using the Adobe Experience Platform SDK and the Edge Network extension.
+
+**Key Features**
```
Comment (emdobrin) [inline src/pages/documentation/release-notes/index.md:15]: I think we can keep just the description underneath, and drop this

**Review by emdobrin — COMMENTED** [review]
looks good with some small comments

**Review by stcheng-adobe — APPROVED** [review]

**Review by emdobrin — APPROVED** [review]

**Comment by emdobrin** [comment]
@calebk1 we should be good to go with this one too, let us know

**Review by calebk1 — APPROVED** [review]
LGTM!

**Review by praschetan — COMMENTED** [review]
I can't edit the fire directly - @timkimadobe - are you able to reflect the following changes back into this copy?

_With this release, the extension enables a drop-in solution for converting Core track API (trackAction & trackState) generated events into Edge Network events. This extension may be used to easily route existing track calls, that may have been implemented for Adobe Analytics, to Edge Network. The configured Data Collection datastream for the mobile application can define a mapping of the track event's contextdata to an XDM schema using [Data Prep for Data Collection](https://experienceleague.adobe.com/docs/platform-learn/data-collection/edge-network/data-prep.html) to route data to Adobe Analytics, Adobe Experience Platform, or to 3rd party vendors via server-side event forwarding._

**Pull request merged**